src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -278,6 +278,10 @@ def load_arguments(self, _):
         c.argument('tags', tags_type)
         c.argument('resource_group_name', resource_group_name_type, options_list=['--name', '-n', '--resource-group', '-g'])
 
+    with self.argument_context('group update') as c:
+        c.argument('properties_to_add', deprecate_info=c.deprecate(hide=True))
+        c.argument('properties_to_remove', deprecate_info=c.deprecate(hide=True))
```

Comment:
Why are generic update arguments hidden for `az group update` in this PR?

Reply:
Because for the `az group create` command, only the `--tags` can be updated, so `properties_to_add` and `properties_to_remove` are useless for this command.
To avoid confusing customers with these useless parameters, we chose to hide them. Actually, we had an old email thread discussing this issue with @dbradish-microsoft, so we made this modification